doc/latex.rst

```diff
@@ -802,18 +802,18 @@ macros may be significant.
 
     Default: ``{rgb}{0,0,0}`` (black)
 
+.. note::
+
+   Starting with the next colour, and for all others following, the
+   names declared to "color" or "xcolor" are prefixed with "sphinx".
+
```

Comment:
I moved it here because it was not clear to what exactly it was referring too

Reply:
I noticed this note interrupts the description list. It might be better to indent with 4spaces to become the part of description for VerbatimBorderColor.